utils/db/repos.ts

```diff
@@ -170,3 +170,50 @@ export const getRepoConfigByUserAndRepo = async (provider: string, repoName: str
 	}
 	return userRows[0].config;
 }
+
+export const insertRepoConfig = async (userId: string, repoIds: number[]) => {
+	const query = `
+		INSERT INTO repo_config (repo_id, user_id, auto_assign, comment_setting)
+		SELECT id, $1, true, true
+		FROM unnest($2::INT[]) AS t(id)
+		ON CONFLICT DO NOTHING
+		`;
+	const params = [userId, repoIds];
+
+	const queryIsSuccessful = await conn.query(query, params)
+	.then((dbResponse) => {
+		if (dbResponse.rowCount == 0) {
+			return false;
+		}
+		return true;
+	})
+	.catch((err: Error) => {
+		console.error(`[db/insertRepoConfigOnSetup] Could not insert repo config for the repos: ${repoIds}`, { pg_query: query }, err);
+		return false;
+	})
+	return queryIsSuccessful;
+}
+  
+export const removeRepoconfigForInstallId = async (install_id: string, repo_names: string[], provider: string, user_id: string) => {
```

Comment:
the name `repo_names` is not descriptive of what its purpose is here. It seems, from the signature, that these repos will be deleted. Whereas these are the repos that will be preserved.

Reply:
how about `repoNamesToBeRetained`?